src/stories/index.stories.js

```diff
@@ -386,41 +421,205 @@ storiesOf('Components', module)
+                                {
+                                    userFriendlyAddress: 'NQ61 H1EF 8AAJ UC8X E8XX U91E KL97 LLLV 7DRH',
+                                    label: 'Not my Savings',
+                                    balance: 7463341234,
+                                },
+                            ],
+                        },
+                        {
+                            id: 'helloworld4',
+                            label: 'Ledger Wallet',
+                            type: 1, // BIP39
+                            accounts: [
+                                {
+                                    userFriendlyAddress: 'NQ55 VDTM 6PVTN672 SECN JKVD 9KE4 SD91 PCCM',
+                                    label: 'Standard Account',
+                                    balance: 123023110,
+                                },
+                                {
+                                    userFriendlyAddress: 'NQ33 DH76 PHUKJ41Q LX3A U4E0 M0BM QJH9 QQL1',
+                                    label: 'My Savings',
+                                    balance: 293872343,
+                                },
+                                {
+                                    userFriendlyAddress: 'NQ21 YPRN 1KVN BQP5 A17U YGD3 HH96 6TKA 6BL4',
+                                    label: 'Standard Account',
+                                    balance: 2023110,
+                                },
+                            ],
+                        },
+                    ],
+                    amount: 199900000,
+                    fee: 138,
+                    shopAddress: 'NQ21 YPRN 1KVN BQP5 A17U YGD3 HH96 6TKA 6BL4',
+                    origin: 'https://shop.nimiq.com',
+                    shopLogoUrl: 'https://shop.nimiq.com/wp-content/uploads/2018/10/nimiq_signet_rgb_base_size.576px.png',
+                    showMerchantInfo: false,
                 };
             },
-            template: windowTemplate(`
-<PaymentInfoLine :amount="199000" :networkFee="1000" :networkFeeEditable="false" origin="https://shop.nimiq.com"/>
-<small-page>
-    <AccountSelector @account-selected="accountSelected" @switch-wallet="switchWallet" @back="back" :accounts="accounts" :walletId="walletId" :walletLabel="walletLabel" :walletType="walletType"/>
+            template: windowTemplate(`<small-page style="position: relative;">
+    <PaymentInfoLine :amount="amount" :fee="fee" :address="shopAddress" :origin="origin" :shopLogoUrl="shopLogoUrl" @merchant-info-clicked="openMerchantInfo"/>
+    <h1 style="font-size: 3rem; text-align: center; margin: 3rem 0 1rem; line-height: 1;">Choose an account to pay</h1>
+    <AccountSelector @account-selected="accountSelected" @login="login" :wallets="wallets" :minBalance="amount + fee"/>
+    <AccountInfo v-if="showMerchantInfo" :address="shopAddress" :origin="origin" :shopLogoUrl="shopLogoUrl" @close="closeMerchantInfo" style="position: absolute; left: 0; top: 0;"/>
```

Comment:
better to make the other story configurable with knobs or a button in the template